pvsreact/src/Components/Header.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import styles from "../CSS/HomeStyle.module.css"
+import logo from "../images/logo.png"
+import { FaUser } from "react-icons/fa";
+import axios from "axios";
+export default function Header({ token, decode }) {
+    const [name, setName] = useState("");
+    useEffect(() => {
+        fetch(`http://localhost:8080/api/getUserByEmail`, {
+            method: 'POST',
+            headers: {
+                'Authorization': `Bearer ${token}`,
+                "Accept": 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                email: decode.sub,
+            }),
+        })
+            .then(response => response.json())
+            .then(data => {
+                console.log(data);
```

Comment:
make sure to remove the console log after merging and testing.

Reply:
removed done.